docs/en-US/component/select-v2.md

```diff
@@ -159,7 +159,7 @@ select-v2/remote-search
 | remote-method                     | function that gets called when the input value changes. Its parameter is the current input value. To use this, `filterable` must be true | function(keyword: string)          | —                   | —             |
 
 <span style="display: none;">
-<!-- | default-first-option | 在输入框按下回车，选择第一个匹配项。需配合 `filterable` 或 `remote` 使用 | boolean | - | false |
+<!-- | default-first-option | 在输入框按下回车，选择第一个匹配项. 需配合 `filterable` 或 `remote` 使用 | boolean | - | false |
```

Comment:
It doesn't seem to need to be changed here.

Reply:
Thanks for the review. The part has been restored to its original state.